string dimension
float64 min_value
float64 max_value
float64 slicing_resolution
lvr_ros/Mesh mesh
float64 grid_resolution
float64 grid_unoccupied_default
std_msgs/ColorRGBA color
---
nav_msgs/OccupancyGrid grid
visualization_msgs/MarkerArray marker
